homeassistant/components/roborock/select.py

```diff
@@ -73,7 +73,9 @@ class RoborockSelectDescription(SelectEntityDescription):
         key="dust_collection_mode",
         translation_key="dust_collection_mode",
         api_command=RoborockCommand.SET_DUST_COLLECTION_MODE,
-        value_fn=lambda api: api.dust_collection_mode.mode.name,  # type: ignore[union-attr]
+        value_fn=lambda api: (
+            mode.name if (mode := api.dust_collection_mode.mode) is not None else None  # type: ignore[union-attr]
+        ),
```

Comment:
Small tweak - I think the policy for multiline lambdas is to have the whole thing wrapped in a brackets.
```suggestion
        value_fn=(
            lambda api: mode.name
            if (mode := api.dust_collection_mode.mode) is not None  # type: ignore[union-attr]
            else None
        ),
```